Formula/l/lima.rb

```diff
@@ -22,12 +22,25 @@ class Lima < Formula
     depends_on "qemu"
   end
 
+  # PR https://github.com/lima-vm/lima/pull/3564
+  # For https://github.com/Homebrew/homebrew-core/pull/224206#issuecomment-2896811998
+  # > Minitest::Assertion: Expected
+  # > {"aarch64" => {"location" => "/home/linuxbrew/.linuxbrew/Cellar/lima/1.1.0/share/lima/
+  # > lima-guestagent.Linux-aarch64.gz"}} to include "riscv64".}
```

Comment:
This comment is not clear to me (probably I don't know enough about brew internals). But it could be more helpful if it explains what we are doing here and why.

Reply:
The issue is that Homebrew has so many `/usr/local/share` directories
- https://github.com/lima-vm/lima/issues/3562#issuecomment-2896814004

The patch introduces a helper script to resolve the `share` path for the additional guest agents.

The patch will not be needed as soon as the PR is merged.

Reply:
The patch should no longer be needed